docs/src/components/code-block/index.tsx

```diff
@@ -6,15 +6,24 @@ import { Language } from 'prism-react-renderer'
 import { ExamplesSwitcher, Example, parseCode } from './examples'
 import styles from './styles.module.css'
 
+interface CodeBlockContextValue {
+  language: Language
+  noRender: boolean
+  startExpanded: boolean
+}
+export const CodeBlockContext = React.createContext<CodeBlockContextValue>({
+  language: 'typescript',
```

Comment:
i'm on the fence about defaulting to typescript...thoughts on defaulting to undefined

Reply:
Yeah, I'm not sure what to default to either.  Why not typescript?

Reply:
One reason not to do undefined is that prism-react-rendered#Highlight requires a Language, at least in typings. Have some default from the list would keep typings easy/correct.

```
  type Language =
    | "markup"
    | "bash"
    | "clike"
    | "c"
    | "cpp"
    | "css"
    | "javascript"
    | "jsx"
    | "coffeescript"
    | "actionscript"
    | "css-extr"
    | "diff"
    | "git"
    | "go"
    | "graphql"
    | "handlebars"
    | "json"
    | "less"
    | "makefile"
    | "markdown"
    | "objectivec"
    | "ocaml"
    | "python"
    | "reason"
    | "sass"
    | "scss"
    | "sql"
    | "stylus"
    | "tsx"
    | "typescript"
    | "wasm"
    | "yaml";
```

Reply:
ah....i see. well i don't know then. typescript seems fine